contrib/devtools/pixie.py

```diff
@@ -0,0 +1,323 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 Wladimir J. van der Laan
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+'''
+Compact, self-contained ELF implementation for bitcoin-core security checks.
+'''
+import struct
+import types
+from typing import Dict, List, Optional, Union, Tuple
+
+# you can find all these values in elf.h
+EI_NIDENT = 16
```

Comment:
note: Now that we require Python 3.6, if you want you can also use [variable annotations](https://www.python.org/dev/peps/pep-0526/):
```suggestion
EI_NIDENT: int = 16
```

Reply:
Sure but, I added any annotations that `mypy` required of me even with various `--disallow-*` options. I'm not convinced that decorating unambiguous assignments of integers with `: int` adds anything but verbosity.